recipes/python-wand/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.5.5" %}
+{% set hash_value = "c97029751f595d96ae0042aec0e26ff114e403e060ae2481124abbcca0c65ce2" %}
+
+package:
+  name: python-wand
```

Comment:
Why `python-wand`? Is there another library called `wand`?

Reply:
I've changed to wand

Reply:
@isuruf, should I also change the recipe folder to wand or leave it as python-wand?